### ISSUE TITLE
svg support ? 

### ISSUE THREAD

**Issue by selul** [DarthSim/imgproxy#112]
Do you think will make sense to add SVG between the list of supported formats? 

Maybe just minify and serve them by default when no output format is requested.

**Comment by DarthSim** [comment]
Hi,

It is possible to use SVG only as a source image. It doesn't require much work on resizing since it can be rendered right in the desired size. I've just tried it, and it works perfectly. But I'm not sure if it makes sense to convert vector to raster.

Converting raster to vector is a very different kind of story. Vectorization is pretty heavy operation and it gives a quite ugly result almost everytime. So it doesn't make sense to me either.

Processing SVG to SVG imposes many restrictions, most processing options just won't work here.

Summing up, if it's ok to use SVG as a source and rasterize it to the result, I can add this feature to imgproxy.

**Comment by selul** [comment]
Thanks for the input! 

For SVG to SVG, just an idea if you would like to consider, maybe minify the SVG resource using something like https://github.com/tdewolff/minify when showing the image? Indeed, any other processing option related to this action doesn't make too much sense, 
I agree that raster to vector conversion is a different kind of story, but from my point of view, the vector to raster will worth to have in order to add the SVG between the list of supported formats.

**Comment by whitmer** [comment]
+1 for svg -> png/jpeg, this is kept me from being able to use this project

**Comment by DarthSim** [comment]
Hi guys!

Sorry for the delay, I was on vacation and took a rest from imgproxy too :)

SVG sources support is added to v2.1.4.
